kritis-charts/values.yaml

```diff
@@ -0,0 +1,45 @@
+  certificate: "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tDQpNSUlEdURDQ0FxQ2dBd0lCQWdJSkFONXdwVEFFRGcraU1BMEdDU3FHU0liM0RRRUJDd1VBTUhFeEN6QUpCZ05WDQpCQVlUQW5oNE1Rb3dDQVlEVlFRSURBRjRNUW93Q0FZRFZRUUhEQUY0TVFvd0NBWURWUVFLREFGNE1Rb3dDQVlEDQpWUVFMREFGNE1Rc3dDUVlEVlFRRERBSmpZVEVsTUNNR0NTcUdTSWIzRFFFSkFSWVdhM0pwZEdsekxYUmxZVzFBDQpaMjl2WjJ4bExtTnZiVEFlRncweE9EQTNNVEF4TnpVek1UWmFGdzB4T1RBM01UQXhOelV6TVRaYU1IRXhDekFKDQpCZ05WQkFZVEFuaDRNUW93Q0FZRFZRUUlEQUY0TVFvd0NBWURWUVFIREFGNE1Rb3dDQVlEVlFRS0RBRjRNUW93DQpDQVlEVlFRTERBRjRNUXN3Q1FZRFZRUUREQUpqWVRFbE1DTUdDU3FHU0liM0RRRUpBUllXYTNKcGRHbHpMWFJsDQpZVzFBWjI5dloyeGxMbU52YlRDQ0FTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBTlpGDQorS2dZVkdGYkliMGJkaVkwaUhtNWhENVRoVDlyZ0JaOE1sOVZ1ckpSNUtxbDRWbTYrOE04Qy9qdy91SWZYTVRyDQo1QnR6RDBzNTlTTE0xbjh2Q3dETnF6SlJIZnViUHUxWHNzbTNoUnpac2JuVXJwS0JmWklkVnhidEhBcWkyTmowDQpNNHluaHpXSjZQdG02Mzl5YVlrbVBGWCtaVVRqT1NxTUpJbUVIck1MbThYVGhrNFpmazFrNHpvSjZoRlFzNW5MDQpONUM0OVVLbG1YQ3BDMTFDNGFQaEJDMXNqYW9qVDE3dXE0bVF4V2RGWDlURXI3WnZlUUUrMjJRaUJXTERnRlU1DQpoMVJYNHhUcUN2NDFsWUVQOFpkTFJUTnpWNFFGay9JVG1TbWRhQ3pFcUxlZk1jeW56ZnNNcW5wanA1S1VjUDdYDQpFekhuN2dSU2kwaGd5aU9TeTkwQ0F3RUFBYU5UTUZFd0hRWURWUjBPQkJZRUZJQVB1UFc1S2F2OCtNblNFQzFnDQpGS2E3YWdPWE1COEdBMVVkSXdRWU1CYUFGSUFQdVBXNUthdjgrTW5TRUMxZ0ZLYTdhZ09YTUE4R0ExVWRFd0VCDQovd1FGTUFNQkFmOHdEUVlKS29aSWh2Y05BUUVMQlFBRGdnRUJBQUU0dStTZUNiSU95SmYycy9NekJvdno2ZlJFDQpKV1B5dkZ5cjY4Q29Ydm9pRnRaVGMvK3crYWdCcTJSTVYzQTVDd2s4RFZLYkk3QmZnNHhxVVZ5Y3hVRFl4d1pUDQpmY3BmZzUxZWVybnZ5YUM0d3lRSGdTMUs1aHNqd1JweURnR285YUZpTkl5QlZvWUhoYjAra05CZlFRbS90OXRqDQpCNFJRaFd2SUIxVWNaV2ZDNlNyRmQ2RlFWRERNT1dLRDVoRUdxdCt6eGVVaWppM2RYemlac0ZXY3JNUlhWbVhYDQpzVG5vaXZXMi9MY2pnSWRxclJTL0RzTGlUc3hCazRzS3Jnd1lzb1VaZThWU2JRUU5sb3VmcDZFWUptSVNlc3Q1DQp6SEpvZnVjQWF4VWExQmIwMHhqcFdiMVMyYVJYQmNUTFZHd3BQbDdMYm80emZRTDV5WjN5bUdGNnpPZz0NCi0tLS0tRU5EIENFUlRJRklDQVRFLS0tLS0="
+
+servicename: kritis-validation-hook
+tlssecretname: tls-webhook-secret
+
+image:
+  repository: gcr.io/kritis-int-test/kritis-server
```

Comment:
s/kritis-int-test/krits-project?  not sure, think it might be kritis-project here

Reply:
yes. the storage bucket for project "kritis-project" was not public, so i added had to push it to kritis-int-test.
Its now public and i will switch it to kritis-project everywhere